### DISCUSSION TITLE
Forms with long content

### DISCUSSION THREAD

**Discussion by PatriRoman** [Telefonica/mistica-design#568] [Proposals]
En el CMS se necesita una solución para poder mostrar al usuario textos largos dentro de forms. Textos que sean más largos que la propia caja. En este caso concreto, un nombre muy largo de una audiencia seleccionada.

Una posible solución que planteo puede ser el uso del tooltip, lo que pasa que ahora mismo en desarrollo su comportamiento es con click (y dentro de un form no podemos poner tooltip con click, evidentemente). 

¿Qué os parece la solución de tooltip con hover? ¿Se os ocurre otra manera de mostrar todo el texto?

<img width="695" alt="image" src="https://user-images.githubusercontent.com/108683238/177182494-0607aa3a-c701-494b-a968-872215c93748.png">

**Comment by yceballost** [comment]
Ticket creado a mistica-web para que evolucione el componente y cambie el evento `on click` por el `hover`
[WEB-599](https://jira.tid.es/browse/WEB-599)

De esta manera por defecto el input no tendría este comportamiento y sería algo "añadido" que en este producto se implementaría. 

Esto es porque creemos que tiene sentido lo que buscáis pero no vemos necesario integrar este comportamiento en el select cuando es un componente que desplegando ya ves la info, el beneficio es mínimo _ioho_, si en un futuro detectaramos que esto es algo muy necesario a incorporar directamente en el input es algo que podríamos valorar.
